### DISCUSSION TITLE
[SUGGESTION] LUA Script Catalogue

### DISCUSSION THREAD

**Discussion by ghost** [azerothcore/azerothcore-wotlk#3937] [Ideas]
Create a catalogue, a bit like the module catalogue but for LUA scripts.

Complete and useful LUA scripts, such as systems of the type (EXP Rates Modifier etc.)

This would highlight the creations of LUA developers and avoid having to share links everywhere to external githubs.

All updated scripts would be centralized and we would know where to look for a lua script.

With always the same prerequisites as for the modules, update your script, make a complete description, screenshots etc.

If this is implemented, it might be interesting to set up a "standard" for using databases when storing data via the LUA.

Why not add a database when the Eluna module is in core, dedicated to LUA data storage.

For example I create an EXP rates modifier, I will use a DB dedicated to Eluna, create a table named mod_exp_modify and do not pollute the characters DB which could avoid some problems when updating the db's.

It needs to be optimized, but the idea is there.

**Comment by Helias** [comment]
The Lua Script are now in the catalogue :+1: 
https://www.azerothcore.org/catalogue/
